feconf.py

```diff
@@ -878,22 +878,22 @@ def get_empty_ratings():
     'RTE_TYPE_TEXTANGULAR': {
         # Valid parent-child relation in TextAngular.
         'ALLOWED_PARENT_LIST': {
-            'p': ['blockquote', 'div', 'pre', '[document]'],
+            'p': ['blockquote', 'div', 'pre', '[document]', 'ol', 'ul', 'li'],
```

Comment:
Just to check, why was this missed before / why did it change?

Reply:
There is a bit problem with the TextAngular Editor after pressing multiple times the list button and shift+enter I did found that `p` can reside in `ol` tag. Even I myself don't remember the exact pattern :P 

Here is the screenshot I took : 
![screenshot from 2018-06-06 01-09-46](https://user-images.githubusercontent.com/22434689/41069859-53261704-6a0d-11e8-99d6-e3d6c9966cd6.png)

Reply:
Oh, interesting! I think that's actually invalid (maybe worth filing a bug on textAngular's repo) according to the box at the top of https://developer.mozilla.org/en-US/docs/Web/HTML/Element/ol -- but since textAngular accepts it, I guess we have to as well :)